bring widgets up to date with Cloud UI (#2344)
elastic/docs: shared/cloud/ess-getting-started-obs.asciidoc

// Include this file in your docs:
// include::{docs-root}/shared/cloud/ess-getting-started-obs.asciidoc[]

. {ess-trial}[Get a free trial].

. Log into {ess-console}[{ecloud}].

. Click *Create deployment*.

. Give your deployment a name.

> . Click *Create deployment*

Reviewer: ```suggestion
. Click *Create deployment*.
```

> . Save your deployment credentials

Reviewer: ```suggestion
. Save your deployment credentials.
```

> You can find your Cloud ID and your APM endpoint in your deployments
> list at {ess-deployments}[cloud.elastic.co], and reset the provided
> password on the *Security* page for your deployment.

Reviewer 2: ```suggestion
You can find your Cloud ID and APM endpoint in your deployments
list at {ess-deployments}[cloud.elastic.co]. To reset the provided
password, go to the *Security* page for your deployment.
```

Reviewer 2: Let's break this into two sentences to avoid putting a comma between the subject and verb that appears in the second independent clause.
